Makefile

```diff
@@ -52,7 +63,9 @@ test-coverage-html:
 	go tool cover -html .cover/cover.out
 
 dist: ## Package up everything in static/ using go-bindata-assetfs so it can be served by a single binary
-	rm -f server/static/bindata_assetfs.go && go-bindata-assetfs -pkg static -prefix server server/static/... && mv bindata_assetfs.go server/static
+	rm -f server/static/bindata_assetfs.go && \
+	go-bindata-assetfs -o server/static/bindata_assetfs.go -pkg static -prefix server server/static/... && \
+	go fmt server/static/bindata_assetfs.go
```

Comment:
I'm not sure, but when I was building this manually I got the impression that there was a reason they did not run `go fmt` on `bindata_assetfs.go`. Possibly because it has huge binary constants representing the compressed assets and it makes `diff`s extra hard to read.

Reply:
it does not matter.
Works in both cases.